code/modules/client/preferences_savefile.dm

```diff
@@ -265,6 +268,7 @@
 
 	WRITE_FILE(S["mute_self_combat_messages"], mute_self_combat_messages)
 	WRITE_FILE(S["mute_others_combat_messages"], mute_others_combat_messages)
+	WRITE_FILE(S["mute_xeno_health_alert_messages"], mute_xeno_health_alert_messages)
```

Comment:
You're changing the preferences save file, make sure to increment the savefile version define.